### DISCUSSION TITLE
Rename of master branch to main in github

### DISCUSSION THREAD

**Discussion by galak** [zephyrproject-rtos/zephyr#35230] [Announcements]
As part of our inclusive language efforts we are changing the development branch in git from `master` to `main`.

You will need to utilize at least west 0.10.1.  If you have an existing checkout of zephyr you'll need to make the following changes:

```
	git branch -m master main
	git fetch origin
	git branch -u origin/main main
```

**Comment by mbolivar-nordic** [comment]
> You will need to utilize at least west 0.10.1.

To expand on this a bit: this is required for plain `west init` to work to initialize a workspace.

You can still specify `west init --mr main` with older versions of west after this change.

**Comment by galak** [comment]
You may also want to make this change in your fork of Zephyr on GitHub.

You can find details on how to do that here:

https://github.com/github/renaming

**Comment by nandojve** [comment]
Can I assume this will be last modification before next tag release?
